package/MDAnalysis/core/groups.py

```diff
@@ -2,7 +2,8 @@
 # vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
 #
 # MDAnalysis --- http://www.MDAnalysis.org
-# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver Beckstein
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver
```

Comment:
No idea. I can remove this one change though. But we have this mixed format in almost all the files already. Everytime I changed the formatting in some random file.